ui/src/theme.js

```diff
@@ -369,6 +386,18 @@ const createTheme = isDarkTheme => {
           },
         },
       },
+      MuiDialogActions: {
+        root: {
+          '& button': {
+            color: isDarkTheme ? 'white' : '#000',
+            '&:hover': {
+              backgroundColor: isDarkTheme
+                ? THEME.TEN_PERCENT_WHITE
+                : THEME.TEN_PERCENT_BLACK,
+            },
+          },
+        },
+      },
       MuiDrawer: {
         paper: {
           maxWidth: 500,
```

Comment:
Hovering on a date removes the background color of the date picker. Hovering on a datepicker in https://material-ui-pickers.dev/demo/datepicker does not remove the background color. Is there a way to fix this?

Reply:
Yes, there is)